sdcm/tester.py

```diff
@@ -736,14 +736,7 @@ def init_resources(self, loader_info=None, db_info=None,
             self.get_cluster_baremetal()
 
         seeds_num = self.params.get('seeds_num', default=1)
-        if self.params.get('instance_provision') == 'mixed':
-            for node in self.db_cluster.nodes:
-                if seeds_num == 0:
-                    break
-                if node.is_spot:
-                    node.is_seed = True
-                    seeds_num -= 1
-        else:
+        if cluster_backend not in ['aws', 'aws-siren']:
```

Comment:
Why you remove code for mode mixed? @bentsi  do we support it or not? 
what happened if configuration for aws backend wil be set to more than 1 seed?

Reply:
`mixed` has a serious issue so can't be used now. The issue is since we use reflector to set seed nodes and reflector itself uses aws instance launch-index to decide which are seeds it cause a problem in mixed mode, since we provison usinf two different requests: one to  create on-demand instances and second spot, instances that we get as a result from running two requests have same launch index so it confuses reflector and it decides that we have two different clusters.

Reply:
I removed the mixed code, because it doesn't work as expected.
After removed it, the is_seed flag will always to be correct.

I think seed_num should not works for aws.

Reply:
Seeds_num was created for ScaleTest which is in AWS.